test/SlowTests/Server/Documents/QueueSink/QueueSinkTestBase.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.QueueSink;
+using Raven.Server.Config;
+using Raven.Server.Config.Categories;
+using Raven.Server.NotificationCenter;
+using Raven.Server.ServerWide;
+using Sparrow.Json;
+using Xunit;
+using Xunit.Abstractions;
+using QueueSinkConfiguration = Raven.Client.Documents.Operations.QueueSink.QueueSinkConfiguration;
+
+namespace SlowTests.Server.Documents.QueueSink
+{
+    [Trait("Category", "QueueSink")]
+    public abstract class QueueSinkTestBase : RavenTestBase
+    {
+        private DocumentStore _src;
+
+        protected static readonly BackupConfiguration DefaultBackupConfiguration;
+
+        static QueueSinkTestBase()
+        {
+            var configuration = RavenConfiguration.CreateForTesting("foo", ResourceType.Database);
+            configuration.Initialize();
+
+            DefaultBackupConfiguration = configuration.Backup;
+        }
+        protected QueueSinkTestBase(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        protected AddQueueSinkOperationResult AddQueueSink<T>(DocumentStore src, QueueSinkConfiguration configuration, T connectionString) where T : ConnectionString
+        {
+            var putResult = src.Maintenance.Send(new PutConnectionStringOperation<T>(connectionString));
+            Assert.NotNull(putResult.RaftCommandIndex);
+
+            var addResult = src.Maintenance.Send(new AddQueueSinkOperation<T>(configuration));
+            return addResult;
+        }
+
+        private async Task<string[]> GetEtlErrorNotifications(DocumentStore src)
```

Comment:
Let's make sure you don't use "ETL" here

Reply:
fixed